### PR TITLE
[DOC-128] fine grained metrics

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /preview/yugabyte-platform/alerts-monitoring/anywhere-metrics/

Build preview : https://deploy-preview-19877--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics/#view-fine-grained-metrics

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md [hunk 1]
```diff
@@ -205,6 +205,12 @@ DocDB uses a highly customized version of [RocksDB](http://rocksdb.org/), a log-
 | Transaction                 | Expired Transactions: the number of expired distributed transactions.<br>Transaction Conflicts: the number of conflicts detected among uncommitted distributed transactions.<br><br>This is related to the process that resolves conflicts for write transactions. This process reads all intents that could conflict and tries to abort transactions with a lower priority. If a write transaction conflicts with a higher-priority transaction, then an error is returned and this metric is iterated. | Alerting is not required.                                    | ![Transaction](/images/yp/metrics75.png)  |
 | Transaction pool cache hits | Percentage of transaction pool requests fulfilled by the transaction pool cache. | This metric is informational and should not be subject to alerting. You may consider this information while examining alerts on other metrics. | ![Transaction pool cache hits](/images/yp/metrics105.png) |
 
+## View fine grained metrics
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md:208]: Maybe move right up to the top, under the first screenshot. No heading.

File: docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md [hunk 2]
```diff
@@ -205,6 +205,12 @@ DocDB uses a highly customized version of [RocksDB](http://rocksdb.org/), a log-
...
+
+YugabyteDB Anywhere allows you to view granular data points in metric charts by selecting a portion of time range along the y-axis of the graph. To do so, navigate to **Universes > Universe-Name > Metrics** and select a portion of any graph to view the granular results as per the following illustration:
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md:210]:
```suggestion
To zoom in to view granular data points, select a portion of the time range along the y-axis of any chart.
```

Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md:210]: Add how to zoom back out again. I couldn't figure it out.

File: docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md [hunk 3]
```diff
@@ -205,6 +205,12 @@ DocDB uses a highly customized version of [RocksDB](http://rocksdb.org/), a log-
...
+
+![Fine grained metrics](/images/yb-platform/fine-grain-metrics.png)
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md:212]: when taking screenshots, try to size the window to the minimum size that shows what we want to show. There is a bunch of whitespace to the right of the 2 charts

File: docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md (hunk 2 above)
Comment (aishwarya24) [reply docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md:210]: You need to click the refresh button. Ill add that and change the screenshot too.

**Comment by ddhodge** [comment]
So there is a subtle way to zoom all charts at once. Should add that.
RE screenshot, actually what I meant was not add a third chart, but narrow the window to the edge of the second chart, given our focus is the first chart...

File: docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md [hunk 4]
```diff
@@ -20,6 +20,12 @@ You access metrics by navigating to **Universes > Universe-Name > Metrics**, as
 
 ![Metrics Page](/images/yp/metrics_main.png)
 
+To zoom in to view granular data points, select a portion of the time range along the y-axis of any chart. Note that all the charts get zoomed in and you can view the detailed metrics.
+
+![Fine grained metrics](/images/yp/fine-grain-metrics.png)
+
+Click the refresh button to zoom out and restore the original metric graphs.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md:27]:
```suggestion
Click the refresh button to restore the default time range.
```

File: docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md [hunk 5]
```diff
@@ -20,6 +20,12 @@ You access metrics by navigating to **Universes > Universe-Name > Metrics**, as
 
 ![Metrics Page](/images/yp/metrics_main.png)
 
+To zoom in to view granular data points, select a portion of the time range along the y-axis of any chart. Note that all the charts get zoomed in and you can view the detailed metrics.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/alerts-monitoring/anywhere-metrics.md:23]:
```suggestion
Use the time menu to change the range. To zoom in to view granular data points, select a portion of the time range along the y-axis of any chart. This zooms all charts.
```

**Review by ddhodge — APPROVED** [review]

**Pull request merged**